features/facebookImport/test/utils/data-importing.js

```diff
@@ -104,38 +96,35 @@ export async function runMessagesImporter(zipFile) {
 }
 
 export async function runRecentlyViewedAdsImporter(zipFile) {
-    return runSingleOutdatedImporter(RecentlyViewedAdsImporter, zipFile);
+    return runSingleImporter(RecentlyViewedAdsImporter, zipFile);
```

Comment:
Wait. No longer outdated?

Reply:
Well, how I understand it, now they stop messing with the data account object and return the expected result, so they are no longer following the old framework.